### PR TITLE
feat: Add BannerText to homepage

### PR DESCRIPTION
**Pull request by danzanzini**
## What's the purpose of this pull request?
To add the [BannerText](https://www.figma.com/file/cAXtiQGObWxamFVVhviQVM/Handoff-Homepage-v0.2?node-id=572%3A57357) to base.store's homepage.

## How it works? 
Just created a new BannerText component inside the sections folder that is a wrapper for [Faststore's Banner component](https://faststoreui.netlify.app/?path=/story/molecules-banner--banner).

## How to test it?
Just go to the preview and see the banner (not-styled) at the homepage

### CODE REVIEW

File: src/components/sections/BannerText/BannerText.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  /**
+   * The content for the p tag below the h2.
+   */
+  caption: string
+
+  /**
+   * The href used at the link
+   */
+  action_path: string
```
Comment (ArthurTriis1) [inline src/components/sections/BannerText/BannerText.tsx:19]: We use camelCase in component props

**Review by ArthurTriis1 — CHANGES_REQUESTED** [review]

File: src/components/sections/BannerText/BannerText.tsx (hunk 1 above)
Comment (danzanzini) [reply src/components/sections/BannerText/BannerText.tsx:19]: 🤣

Comment (filipewl) [reply src/components/sections/BannerText/BannerText.tsx]: _Ruby feelings!_

Comment (danzanzini) [reply src/components/sections/BannerText/BannerText.tsx]: I got nostalgic with the launch of Rails 7

**Review by filipewl — APPROVED** [review]
This looks good enough for me right now.

If it becomes a need in the future, we can give users more flexibility by allowing the props to be `ReactNode` instead of just `string` (I guess `ReactNode` also accepts strings) so we can unify props like `action={(<a href="/">Click here</a>)}`, or allow `title` and `caption` to have links within (not possible with strings?).

**Comment by hellofanny** [comment]
Uhull! LGTM, but not sure if we should replace the `BannerLink` for the `Link` already 👀

File: src/components/sections/BannerText/BannerText.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+}: BannerTextProps) {
+  return (
+    <Banner>
+      <BannerContent>
+        <div>
+          <h2>{title}</h2>
+          <p>{caption}</p>
+        </div>
+        <BannerLink>
+          <a href={actionPath}>{actionLabel}</a>
```
Comment (tlgimenes) [inline src/components/sections/BannerText/BannerText.tsx:41]: Please, use Gatsby's Link for client-side navigation

**Review by tlgimenes — CHANGES_REQUESTED** [review]

**Review by lucasfp13 — APPROVED** [review]

**Review by eduardoformiga — APPROVED** [review]

**Review by hellofanny — APPROVED** [review]

**Review by saranicoly — APPROVED** [review]

**Pull request merged**